CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ endif()
 
 enable_testing()
 add_subdirectory( test )
-add_subdirectory( benchmark )
+#add_subdirectory( benchmark )
```

Comment:
Why was this commented out?

Reply:
I see your PR comment, add that as a comment in the CMakeLists.txt

Reply:
alternatively you could `EXCLUDE_FROM_ALL` here

Reply:
> alternatively you could `EXCLUDE_FROM_ALL` here

Thanks Matt, much nicer, done.